README.md

```diff
@@ -13,3 +13,13 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+1. Asymptotic analysis just measures how it grows not the actual speed. Therefore constants are ignored and those matter in pratice.
+2. It also doesn't consider space, in the real world using more space is expensive both financially and speed wise as data is further and further away physically.
+3. The best case and average and worst case might have varying levels of occurance. For example the best case of insertion sort is faster than the best case for quick sort, but the best case for quick sort occurs far more often so on average it is faster.
+
+- The complexity of BST is O(log n) as the input increases by a factor of 10 I'd expect the time to increase by a factor of log 10 which is 3.322. So around 16 seconds roughtly.
+
+- It could be that the new items were sorted so the tree ended up with a very large height and was not balanced. Meaning that the search time was closer to that of a linked list than a BST.
```

Comment:
You can't take the log of the factor -- the complexity applies to the runtimes, not ratios of runtimes.